### ISSUE TITLE
Not able to fetch substring

### ISSUE THREAD

**Issue by gagan-bansal** [nolanlawson/substring-trie#3]
Not able to fetch substring, here is use case
```javascript
var Trie = require('substring-trie')
var trie = new Trie(['banana', 'grape', 'grape fruit', 'grape fruit sweet']
// 'grape fruit sour' return nothing while expected is 'grape fruit'
trie.search('grape fruit sour') 
```

**Comment by nolanlawson** [comment]
Thanks for reporting; this is now fixed in v1.0.2. :smiley:

**Comment by gagan-bansal** [comment]
Great!, thanks for update.
